libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -500,14 +506,15 @@ void AP_InertialSensor_Backend::update_gyro(uint8_t instance)
     }
 
     // possibly update filter frequency
-    if (_last_gyro_filter_hz != _gyro_filter_cutoff()) {
+    if (_last_gyro_filter_hz != _gyro_filter_cutoff() || sensors_converging()) {
```

Comment:
just checking that the intention here is that the lines below (set_cutoff_frequency, etc) are run all the time during the first 30 seconds that the board is powered up?

Reply:
@rmackay9 see my comment https://github.com/ArduPilot/ardupilot/pull/11795#issuecomment-526485212 - during this "warmup" phase we can change the sample frequency - this is designed to catch that. Whether this is the right way to warmup is another question but it works and is consistent with the way we are changing the sample frequency.